src/about.html

```diff
@@ -15,6 +15,23 @@
   </head>
   <body>
     <header>
+      <div class="topBar">
+        <ul class="pages">
+          <li>
+            <a href="#">
+              <i class="fab fa-facebook"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
       <nav class="navbar navbar-expand-lg navbar-light bg-white fixed-top">
         <div class="container-fluid container">
           <a class="navbar-brand" href="#">
```

Comment:
- [ ] dear @bekjon-me, Just as mentioned by the previous reviewer,  The About page is in the middle section with a central column that includes a section header, a divider, a paragraph of text, and an image or large logo. But in the case of your about page, you miss the divider, a paragraph of text, and an image or large logo.

### Given design
![image](https://user-images.githubusercontent.com/69077061/219755539-196f8773-0c76-4ada-a75c-d2b6b71cd6d5.png)
Your design as of now
![image](https://user-images.githubusercontent.com/69077061/219755738-14e2c123-2a06-425d-a27c-5890c6683d6e.png)

- [ ] Also, as mentioned by the previous reviewer we are missing a section here, which is on the About page a bottom section with a central column that includes a section with two images with text centered over them just like the one below.
![219149281-f6c6b426-ee79-4086-8c20-c1118c640c70](https://user-images.githubusercontent.com/69077061/219757011-101256e0-4ba4-4d84-9a6f-5ca4b5424a09.png)

Reply:
- [ ] Don't get tired. I see that you haven't made any attempt to work on these recommended sections. Kindly be patient and approach. Do not forget that you have request limits. So, let's save some cards and just carefully fix this. Again, take a deep breath to ease the tension. :love_you_gesture: